### PR TITLE
1h sword aserai edits

### PR DESCRIPTION
**Pull request by Truthful33**
(FIX) Edited the following items. Fixed stab values based on feedback:

- crpg_aserai_noble_sword_1_t5 | Decorated Kaskara
- crpg_aserai_noble_sword_2_t5 | Decorated Long Scimitar
- crpg_aserai_noble_sword_3_t5 | Decorated Scimitar with Wide Grip
- crpg_aserai_noble_sword_4_t5 | Engraved Angular Kaskara
- crpg_aserai_sword_1_t2 | Iron Flyssa
- crpg_aserai_sword_2_t2 | Iron Scimitar
- crpg_aserai_sword_3_t3 | Slightly Ridged Flyssa
- crpg_aserai_sword_4_t4 | Fine Steel Long Kaskara
- crpg_aserai_sword_5_t4 | Long Fine Steel Scimitar
- crpg_aserai_sword_6_t4 | Narrow Fine Steel Kaskara
- crpg_aserai_sword_7_t5 | Thamaskene Steel Kaskara

Find additional details [here](https://docs.google.com/spreadsheets/d/1sTt6PEGD3EHn-EBzABQRszbdwDkoSAa765WSm8Vb5JE/edit#gid=1341011737) (Aserai Changes).

### CODE REVIEW

File: crafting_pieces.xml [hunk 1]
```diff
@@ -4942,10 +4941,10 @@
       <Material id="Iron6" count="3" />
     </Materials>
   </CraftingPiece>
-  <CraftingPiece id="crpg_aserai_noble_sword_1_t5_blade" name="{=gwkR6TZL}Decorated Kaskara Blade" tier="5" piece_type="Blade" mesh="aserai_noble_blade_2" culture="Culture.aserai" length="93.2" weight="1.0">
+  <CraftingPiece id="crpg_aserai_noble_sword_1_t5_blade" name="{=gwkR6TZL}Decorated Kaskara Blade" tier="5" piece_type="Blade" mesh="aserai_noble_blade_2" culture="Culture.aserai" length="93.2" weight=".7">
```
Comment (namidaka) [inline crafting_pieces.xml:4944]: .7 -> 0.7

File: crafting_pieces.xml [hunk 2]
```diff
@@ -4930,10 +4930,9 @@
       <Material id="Iron6" count="2" />
     </Materials>
   </CraftingPiece>
-  <CraftingPiece id="crpg_aserai_noble_sword_2_t5_blade" name="{=T1xFTaOf}Decorated Long Scimitar Blade" tier="5" piece_type="Blade" mesh="aserai_noble_blade_1" culture="Culture.aserai" length="90" weight="1.05">
+  <CraftingPiece id="crpg_aserai_noble_sword_2_t5_blade" name="{=T1xFTaOf}Decorated Long Scimitar Blade" tier="5" piece_type="Blade" mesh="aserai_noble_blade_1" culture="Culture.aserai" length="90" weight=".75" excluded_item_usage_features="thrust">
```
Comment (namidaka) [inline crafting_pieces.xml:4933]: .75 -> 0.75

File: crafting_pieces.xml [hunk 3]
```diff
@@ -4698,10 +4698,10 @@
       <Material id="Iron6" count="3" />
     </Materials>
   </CraftingPiece>
-  <CraftingPiece id="crpg_aserai_noble_sword_4_t5_blade" name="{=RdVX1jC0}Engraved Kaskara Blade" tier="5" piece_type="Blade" mesh="aserai_noble_blade_3" culture="Culture.aserai" length="71" weight="1.08">
+  <CraftingPiece id="crpg_aserai_noble_sword_4_t5_blade" name="{=RdVX1jC0}Engraved Kaskara Blade" tier="5" piece_type="Blade" mesh="aserai_noble_blade_3" culture="Culture.aserai" length="71" weight=".79">
```
Comment (namidaka) [inline crafting_pieces.xml:4701]: .79 -> 0.79

**Review by namidaka — CHANGES_REQUESTED** [review]

**Comment by Truthful33** [comment]
Committed the changes

File: crafting_pieces.xml [hunk 4]
```diff
@@ -4686,10 +4686,10 @@
       <Material id="Iron6" count="3" />
     </Materials>
   </CraftingPiece>
-  <CraftingPiece id="crpg_aserai_noble_sword_3_t5_blade" name="{=xYgJN0Ja}Decorated Scimitar Blade" tier="5" piece_type="Blade" mesh="aserai_noble_blade_4" culture="Culture.aserai" length="87.4" weight="1.0">
+  <CraftingPiece id="crpg_aserai_noble_sword_3_t5_blade" name="{=xYgJN0Ja}Decorated Scimitar Blade" tier="5" piece_type="Blade" mesh="aserai_noble_blade_4" culture="Culture.aserai" length="87.4" weight=".75">
```
Comment (namidaka) [inline crafting_pieces.xml:4689]: .75-> 0.75

**Review by namidaka — CHANGES_REQUESTED** [review]

File: crafting_pieces.xml (hunk 4 above)
Comment (Truthful33) [reply crafting_pieces.xml]: committed

**Review by namidaka — APPROVED** [review]

**Pull request merged**